src/Propel/Generator/Command/InitCommand.php

```diff
@@ -264,7 +264,7 @@ private function reverseEngineerSchema(OutputInterface $output, array $options)
         $outputDir = sys_get_temp_dir();
 
         $this->getApplication()->setAutoExit(false);
-        $fullDsn = sprintf('%s;user=%s;password=%s', $options['dsn'], urlencode($options['user']), urlencode($options['password']));
+        $fullDsn = sprintf('%s;user=%s;password=%s', $options['dsn'], $options['user'], $options['password']);
```

Comment:
@marcj If we refactor some code, we can pull the username/password out of the dsn.  I would be happy to take a stab at that if that is the way you want to go.  That will be a better solution than simply removing urlencode from values to be placed in the dsn string.